### DISCUSSION TITLE
Does WasmEdge support tiny IoT devices?

### DISCUSSION THREAD

**Discussion by TAOFOR4** [WasmEdge/WasmEdge#3343] [Q&A] [answered]
Hi, 

Thx for this brilliant project. I am wondering if WasmEdge supports some tiny IoT devices like ESP32 C6, Raspberry Pi Pico, and so on. I haven't found any documents for that. Thank you!

**Comment by hydai** [comment, marked as answer]
Hi @TAOFOR4 
The short answer is NO. WasmEdge currently doesn't have official support on such platforms. Since WasmEdge is an open-source project, any PR that can port WasmEdge on these platforms is welcome.
